client/src/components/tests/UserInfo1.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Enzyme, { shallow, mount } from 'enzyme';
+
+import UserInfo1 from '../UserInfo1';
+
+describe('UserInfo1', () => {
+  const props = {
+    stockInfo: [
```

Comment:
can do additional testing with some filled out stockInfo and test if it matches in the shallow render.